utils/misc/misc.go

```diff
@@ -420,7 +420,8 @@ func Copy(dst, src interface{}) {
 func GetIPFromReq(req *http.Request) string {
 	addresses := strings.Split(req.Header.Get("X-Forwarded-For"), ",")
 	if addresses[0] == "" {
-		return req.RemoteAddr // When there is no load-balancer
+		splits := strings.Split(req.RemoteAddr, ":")
```

Comment:
When is this happening? Earlier we were getting the right request IP.

This solution will conflict with IP V6 addresses.

Reply:
Joining them back again except the last split ":".
Will it still be an issue with ip v6